src/components/PlaceItem/PlaceItem.tsx

```diff
@@ -0,0 +1,34 @@
+// import Image from "next/image";
+import { Place } from "../../data/memorialPlacesData";
+import { Button } from "../shared/Button/Button";
+
+export const PlaceItem: React.FC<{ miejsce: Place }> = ({ miejsce }) => (
```

Comment:
Use english names for prop names please. Also related to `React.FC<Props>` from previous comment.

It is much simpler to type it like this (like any regular function):
```tsx
export const PlaceItem = ({ place }: { place: Place }) => {};
```
(I use the above inline notation with short props definitions)

Or my personal preference ❤️ :
```tsx
interface Props {
    place: Place;
}

export const PlaceItem = ({ place }: Props) => {}
```
Simple and elegant. Also no need to export Props and to change the interface name to `ComponentNameProps` as it is only used within this file and TS will take care of the rest during components usage.